pkg/kubelet/metrics/metrics.go

```diff
@@ -573,6 +574,17 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// TopologyManagerAdmissionDuration is a Histogram that tracks the duration (in seconds) to serve a pod admission request.
+	TopologyManagerAdmissionDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           TopologyManagerAdmissionDurationKey,
+			Help:           "Duration in milliseconds to serve a pod admission request.",
+			Buckets:        metrics.ExponentialBuckets(.05, 2, 15),
+			StabilityLevel: metrics.ALPHA,
```

Comment:
What does the ALPHA mean here?

Reply:
Similar to feature lifecycle, we have [metric lifecycle](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/#metric-lifecycle) such that a metric has a `StabilityLevel` based on how stable it is. My understanding is that ideally, a metric should graduate when the associated feature graduates. For this PR however, Alpha as the stability level of the metric made sense to me as we are introducing it even though the associated feature i.e. Topology Manager is graduating to GA.